src/media/modules/class-wpml-elementor-media-node-slides.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class WPML_Elementor_Media_Node_Slides extends WPML_Elementor_Media_Node {
```

Comment:
@strategio Slides and Carousel are exactly the same except for the property name. Please remove the duplication.